src/fontra/client/web-components/range-slider.js

```diff
@@ -164,6 +165,17 @@ export class RangeSlider extends html.UnlitElement {
     this.sawMouseDown = false;
     this.sawMouseUp = false;
     this.onChangeCallback = () => {};
+    this.values = [];
+    this._sortedValuesCached = undefined;
+  }
+
+  get valuesSorted() {
```

Comment:
I'm wondering if the order even matters. For a discrete value slider, minValue and maxValue are meaningless and are ignored. For example, if the values were [50, 0, 100], I see no real reason why they could not be represented directly without sorting, as slider values 0, 1 and 2.

Reply:
It's for finding the closest value. We start seeking from the lowest one, iterate until the current one is greater than actual value.

Reply:
Ok, I can see we need sorting for that. It would be nice if it would find the actual closest, for example for list [0, 100], 200 would find 0, and 60 would find 100.